src/NHN.DtoContracts/NHN.DtoContracts/Flr/Data/PrimaryHealthCareContractEdit.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Runtime.Serialization;
+using NHN.DtoContracts.Common.en;
+
+namespace NHN.DtoContracts.Flr.Data
+{
+    /// <summary>
+    /// Basisdata for en person.
```

Comment:
Basis data for en fastlegekontrakt i FLR som er tilknyttet et primærhelseteam. Objekt til bruk ved opprettelse og endring.